src/pyfmi/simulation/assimulo_interface_fmi3.pyx

```diff
@@ -505,7 +506,7 @@ cdef class FMIODE3(cExplicit_Problem):
                 for e in solver.get_weighted_local_errors():
                     diag_data[index] = e
                     index +=1
-            support_event_indicators = (solver_name=="CVode" or solver_name=="Radau5ODE" or solver_name=="LSODAR" or solver_name=="ImplicitEuler")
+            support_event_indicators = (solver_name=="CVode" or solver_name=="Radau5ODE" or solver_name=="ExplicitEuler" or solver_name=="LSODAR" or solver_name=="ImplicitEuler")
```

Comment:
I saw this also in the other PR, I wonder if it would be better if we have a function or an enumeration or list and update this to `support_event_indicators = solver_supports_event_indicators(solver_name)`?
Ideally `solver.supports_event_indicators` but I think that would be out of scope.

Reply:
Sounds good to me, well done!